### ISSUE TITLE
Allow zero gain in augain module

### ISSUE THREAD

**Issue by GitHubIsShortOnUsernames** [baresip/baresip#3223]
Since there is only a mute option for capture, and muting through ALSA generates a clicking sound on some hardware, it would be nice to be able to use augain for muting the output. However, gain is capped at `if (new_gain <= 0.0)` . Could this be changed to `< 0.0`?

**Comment by juha-h** [comment]
I don't have anything against the change, but using augain module for muting sounds like a hack to me.

**Comment by GitHubIsShortOnUsernames** [comment]
Yes, it's a very inefficient way to mute audio, but the easiest option for now. When I have the time to look into it, I'll provide a change request for a proper mute function.

**Comment by cspiel1** [comment]
The baresip core supports TX mute with function `audio_mute()`.

Module menu has a command for toggle mute.

**Comment by NotherGithubAccount** [comment]
I was looking to mute RX. Currently I'm leaning towards using ALSA's softvol plugin, which does not trigger the clicks the hardware mixer does on some audio interfaces. It seems more suitable than abusing augain. ;-)

**Comment by alfredh** [comment]
Can you please provide an example PR for how you want this to work ?

Is the topic of the mute for recording, playback or both ?

Muting recording is already supported today and implemented in the core.

**Comment by GitHubIsShortOnUsernames** [comment]
It was about RX mute in addition to TX. But at this stage I think it is better done on the audio backend, since all of them offer that functionality already. No need to duplicate it in baresip. Problem was ALSA producing clicks when muting, which can be worked around with the softvol ALSA plugin.

At the moment I'm struggling with buffer settings a bit. Could you maybe clarify the audio_buffer and jitter buffer settings in the documentation please? To me it sounds like the audio_buffer isn't needed when there's a jitter buffer, and the alsa module is also setting period size to ptime with 4x ptime as buffer. However, audio_buffer still needs to be at least 1x ptime to work at all, and 2x ptime to prevent dropouts. I don't understand what its purpose is when there should be enough buffered audio available already? Looking to minimize latency ...
